options/locale/locale_en-US.ini

```diff
@@ -922,6 +922,16 @@ settings.protected_branch=Branch Protection
 settings.protected_branch_can_push=Allow push?
 settings.protected_branch_can_push_yes=You can push
 settings.protected_branch_can_push_no=You can not push
+settings.branch_protection = Branch Protection for <b>%s</b>
+settings.protect_this_branch = Protect this branch
+settings.protect_this_branch_desc = Disable force pushes and prevent from deletion.
+settings.update_protect_branch_success = Change branch protect options successfully.
+settings.protect_whitelist_committers = Whitelist who can push to this branch
+settings.protect_whitelist_committers_desc = Add people or teams to whitelist of direct push to this branch. Users in whitelist will bypass require pull request check.
```

Comment:
I would suggest `Add users or teams to this branch's whitelist. Whitelisted users bypass the typical push restrictions.`

Reply:
done.